daos/view.go

```diff
@@ -66,7 +66,8 @@ func (dao *Dao) SaveView(name string, selectQuery string) error {
 		if _, err := txDao.TableInfo(name); err != nil {
 			// manually cleanup previously created view in case the func
 			// is called in a nested transaction and the error is discarded
-			txDao.DeleteView(name)
```

Comment:
If the editor does not give a warning for this type of use, it is not clear that an error message has not been handled.